src/program/snabbnfv/neutron2snabb/neutron2snabb.lua

```diff
@@ -59,8 +59,8 @@ function create_config (input_dir, output_dir, hostname)
                             { vlan = vif_details.zone_vlan,
                               mac_address = port.mac_address,
                               port_id = port.id,
-                              ingress_filter = filter(port, secbindings, secrules, 'ingress'),
-                              egress_filter = filter(port, secbindings, secrules, 'egress'),
+                              ingress_filter = filter(port, secbindings, secrules, 'ingress', vif_details.packetfilter),
+                              egress_filter = filter(port, secbindings, secrules, 'egress', vif_details.packetfilter),
                               gbps = vif_details.zone_gbps,
```

Comment:
> The packetfilter setting should be in profile rather than vif_details.

This line looks a bit fishy too now that I think about it. We never use the `gbps` field in `nfvconfig`... (and some sort of `tx_police_gbps` is missing too).